templates/kobo-env/postgres/primary/postgres.conf.tpl

```diff
@@ -14,3 +14,7 @@
 # Total Memory (RAM): ${POSTGRES_RAM}GB
 
 ${POSTGRES_SETTINGS}
+
+${USE_WAL_E_BACKUP}archive_mode = on
+${USE_WAL_E_BACKUP}archive_command = 'envdir /var/lib/postgresql/data/wal-e.d/env wal-e wal-push %p'
```

Comment:
I think `/var/lib/postgresql/data` could be replaced by `$PGDATA`.

Reply:
Can't use variable directly in postgresql.con file
example:
`archive_command = 'envdir $PGDATA/wal-e.d/env wal-e wal-push% p'`

It works with a sed when starting the docker:
`sed -i '/archive_mode/ a\'"archive_command = 'envdir $PGDATA/wal-e.d/env wal-e wal-push %p'" /var/lib/postgresql/data/postgresql.conf`

Reply:
I got it to work with @noliveleger's suggestion of using an escape character in the `postgres.conf.tpl` file: `${USE_WAL_E_BACKUP}archive_command = 'envdir $$PGDATA/wal-e.d/env wal-e wal-push %p'`

Or are you saying that `$PGDATA` can't actually be used directly in the resulting `postgresql.conf` file?

Reply:
The substitution of `PGDATA` is done in `postgres` container. See https://github.com/kobotoolbox/kobo-docker/pull/310/